src/less/main.module.less

```diff
@@ -0,0 +1,26 @@
+.gridContainer {
```

Comment:
Use dashes and not camel case here (grid-container)

Reply:
Remember to update the html element accordingly

Reply:
Since I am using CSS modules, it is recommended to use camel case for local names since it's less clunky to work with when importing in the code: https://github.com/css-modules/css-modules/blob/master/README.md#naming

I tried kebab case at first and can indeed confirm that it was clunky.